### PR TITLE
Issue270 - add xcuitest for a nonsunscribed vpn user signing in and out

### PR DESCRIPTION
**Pull request by ashrivastava-qa**
@isabelrios please review. Thanks!

### CODE REVIEW

File: FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift [hunk 1]
```diff
@@ -79,4 +79,42 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
         XCTAssertTrue(app.buttons["ReloadButton"].exists)
         XCTAssertTrue(app.webViews.textFields["Email"].exists)
     }
+
+    func testSignInAsNonSubscribedUser() {
+
```
Comment (isabelrios) [inline FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift:84]: You can remove this space

File: FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift [hunk 2]
```diff
@@ -79,4 +79,42 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
...
+        //Enter the password
+        app.secureTextFields["Password"].tap()
+        app.typeText("gkgJqyzJ")
+        app.buttons["Sign in"].tap()
+
+        //Verify that a non subscibed VPN user can see the 'Try Mozilla link' message
+        waitForExistence(app.staticTexts["Mozilla VPN"], timeout: 15)
+        XCTAssertTrue(app.staticTexts["Subscribe to turn on VPN. Try Mozilla VPN"].exists)
+
+       //Sign out
```
Comment (isabelrios) [inline FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift:108]: Here you are going to Settings before signing out. I would modify the comment like // Go to Settings and Sign out, so with one comment is enough and clear.

File: FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift [hunk 3]
```diff
@@ -79,4 +79,42 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
...
+       app.tabBars.buttons["Settings"].tap()
+
+       //Click the signout option
```
Comment (isabelrios) [inline FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift:111]: Update comment above and you can remove this one

File: FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift [hunk 4]
```diff
@@ -79,4 +79,42 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
...
+        app.buttons["Get started"].tap()
+
+        // Wait for the FxASingIn page to be shown
+        waitForExistence(app.webViews.textFields["Email"], timeout: 10)
+        app.textFields["Email"].tap()
+        app.typeText("test-f5aefc1935@restmail.net")
+        waitForExistence(app.buttons["Continue"], timeout: 20)
+        app.buttons["Continue"].firstMatch.tap()
+
+        //Enter the password
```
Comment (isabelrios) [inline FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift:99]: Please add space between // and text

File: FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift [hunk 5]
```diff
@@ -79,4 +79,42 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
...
+        app.secureTextFields["Password"].tap()
+        app.typeText("gkgJqyzJ")
+        app.buttons["Sign in"].tap()
+
+        //Verify that a non subscibed VPN user can see the 'Try Mozilla link' message
```
Comment (isabelrios) [inline FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift:104]: Same, missing space

File: FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift [hunk 6]
```diff
@@ -79,4 +77,40 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
...
+        waitForExistence(app.staticTexts["Mozilla VPN"], timeout: 5)
+        XCTAssertTrue(app.staticTexts["Mozilla VPN"].exists, "The main page is not loaded correctly")
+
+        // Tap on Get started
+        app.buttons["Get started"].tap()
+
+        // Wait for the FxASingIn page to be shown
+        waitForExistence(app.webViews.textFields["Email"], timeout: 10)
+        app.textFields["Email"].tap()
+        app.typeText("test-f5aefc1935@restmail.net")
```
Comment (isabelrios) [inline FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift:92]: Could you create a variable to add this test account, something like:
let nonSubscribedEmailAccount = "test-f5aefc1935@restmail.net" 
Something similar to this [example](https://github.com/mozilla-mobile/firefox-ios/blob/main/XCUITests/SyncFAUITests.swift#L11)

File: FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift [hunk 7]
```diff
@@ -79,4 +77,40 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
...
+        waitForExistence(app.buttons["Continue"], timeout: 20)
+        app.buttons["Continue"].firstMatch.tap()
+
+        // Enter the password
+        app.secureTextFields["Password"].tap()
+        app.typeText("gkgJqyzJ")
```
Comment (isabelrios) [inline FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift:98]: Similar variable added for password would be helpful too

**Review by rlr — APPROVED** [review]
lgtm

**Pull request merged**